docs/sync.js

```diff
@@ -698,7 +698,7 @@ class User {
     /**
      * Calls the named server function as this user.
      * @param {string} name - name of the function to call
-     * @param {any[]} args - list of arguments to pass
+     * @param {any[]} args - list of arguments to pass, if no arguments are passes, `[]` is used
      * @return {Promise<any>} - resolves when the function terminates.
      */
     callFunction(name, args) { }
```

Comment:
Can you do `args = []` here? I prefer putting things like that in code, rather than prose, although the esdoc html generator has some weird limitations. If not, you should probably at least change the type on line 701 to indicate optionality.

Also, we need to fix this since it is extremely wrong... https://github.com/realm/realm-js/blob/364448f3267e4f733018a248fe7d5a370f625a15/types/app.d.ts#L495

Reply:
Just to make this obvious. The types you're referring to are shared with Realm Web.
I've created an issue to track fixing this: https://github.com/realm/realm-js/issues/3733